server/src/test/java/io/crate/integrationtests/StorageOptionsITest.java

```diff
@@ -112,6 +112,7 @@ tsz timestamp with time zone storage with (columnstore=false)
         execute("insert into tbl (ts, tsz) values (?, ?)", new Object[]{"2025-06-27 11:22:33.987", "2025-06-27 11:22:33.987+02:00"});
         execute("refresh table tbl");
         execute("select ts, tsz from tbl limit 1");
-        assertThat(response).hasRows("1751023353987| 1751016153987");
+        assertThat((long) response.rows()[0][0]).isGreaterThan(1750000000000L); // Rough check due to randomized tz
+        assertThat((long) response.rows()[0][1]).isEqualTo(1751016153987L);
```

Comment:
afaik we just want to assert that no exception is thrown and valid records are returned. so we could also just assert > 0. but we can also leave it like that, up to you.
```suggestion
        assertThat((long) response.rows()[0][0]).isGreaterThan(0); // Rough check due to randomized tz
        assertThat((long) response.rows()[0][1]).isEqualTo(1751016153987L);
```